src/protagonist/API/Features/Image/Ingest/AssetProcessor.cs

```diff
@@ -18,27 +19,31 @@ namespace API.Features.Image.Ingest;
 /// </summary>
 public class AssetProcessor
 {
-    private readonly IPolicyRepository policyRepository;
     private readonly IApiAssetRepository assetRepository;
     private readonly IStorageRepository storageRepository;
+    private readonly IDefaultDeliveryChannelRepository defaultDeliveryChannelRepository;
+    private readonly IDeliveryChannelPolicyRepository deliveryChannelPolicyRepository;
     private readonly ApiSettings settings;
+    private const string None = "none";

```

Comment:
Could `AssetDeliveryChannels.None` be used instead of defining the name of the None channel here?

Reply:
This is the name of the policy as opposed to the channel - the name is confusing though, so will make it more clear